### DISCUSSION TITLE
Result

### DISCUSSION THREAD

**Discussion by Lesomebody** [jaykali/maskphish#121] [Q&A] [answered]
How can i look for the result after phising somebody?

**Comment by jaykali** [comment, marked as answer]
Please use it carefully, and be responsible. Don't use this to somebody. We not will be responsible if you damage anyone.

Now come to your question. MaskPhish didn't made for using solo. You just can hide you phishing link with MaskPhish. MaskPhish just hide the link didn't change it. Just change the outfit of the phishing link. So you need to relay on your phishing tool/method for the result.

Thanks
